### DISCUSSION TITLE
706

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#889] [Comments]
# 706

SNS에 글 올렸다가 기록차 + 추가적인 생각이 들어서 블로그에 남깁니다. 우리팀은 AWS ECS Fargate 를 대부분의 애플리케이션에서 사용한다. 그러다보니 예전처럼 로그 관리에 크게 신경을 쓰지 않게 되었다. 모든 출력에 대해 CloudWatch로 연동이 되어있어서 그전까지 해오던 Logback의 Rotate나 Linux의 Log Rotate 등에 대해 전혀 신경쓰지 않는 것이다. CloudWatch에 적재된 로그는 DataDog으로 전달되어 에러 추적, 모니터링, 알람등으로 사용된다. 용량 혹은 시간 기반으로 로테이트 시킬까 INFO, ERROR 등 각 레벨의 로그 파일은 어디에 위치시킬까 서버 용량에 대한 알람을 걸어두어야한다던가 로그를 모니터링 시스템으로 연동하기 위해 어떤 Agent를 어떻..

https://jojoldu.tistory.com/706

**Comment by BokEumEom** [comment]
너무 공감되네요;;;
